to-do-list/.eslintrc.json

```diff
@@ -0,0 +1,31 @@
+{
+  "env": {
+    "browser": true,
+    "es2021": true
+  },
+  "extends": [
+    "plugin:react/recommended",
+    "standard"
+  ],
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "ecmaVersion": 12,
+    "sourceType": "module"
+  },
+  "settings": {
+    "react": {
+      "version": "detect"
+    }
+  },
+  "plugins": [
+    "react"
+  ],
+  "rules": {
+    //Next.js imports React automatically
+    "react/react-in-jsx-scope": "off",
+    //Not using TypeScript
+    "react/prop-types": "off"
+  }
+}
```

Comment:
Please add line at the end of file

Reply:
I added the line and, i committed that request. Let me know if i did it correctly.